apps/src/p5lab/P5LabView.jsx

```diff
@@ -58,11 +58,18 @@ class P5LabView extends React.Component {
     this.props.onMount();
     const locale = window.appOptions.locale;
     const app = this.props.spriteLab ? 'spritelab' : 'gamelab';
-    fetch(`/api/v1/animation-library/manifest/${app}/${locale}`)
-      .then(response => response.json())
-      .then(libraryManifest => {
-        this.setState({libraryManifest});
+    // temporarily pull from local env for testing purposes
```

Comment:
[friendly reminder since we're coming back to this PR after a while] don't forget to remove this test code!